site/lib/_sass/components/_tooltip.scss

```
@@ -0,0 +1,61 @@
+.tooltip-wrapper {
+  position: relative;
+
+  a.tooltip-target {
+    color: inherit;
+    text-decoration: underline dotted;
```

I don't know why, but this seems to not work on Safari unless you separate them:

```suggestion
    text-decoration: underline;
    text-decoration-style: dotted;
```